CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [0.0.next] - 2017-06-02
+### Changed
+- java-heap-pcnt.sh and metrics-java-heap-graphite.sh: removed extra column from 'UsedHeap'
```

Comment:
it does, we just need to make sure that we have appropriate information for users so they can determine if this breaks for them. My quick investigation led me to different versions and columns but it certainly could be more complicated.